### DISCUSSION TITLE
How to only show the file name in the header?

### DISCUSSION THREAD

**Discussion by ccshao** [sharkdp/bat#2383] [Q&A]
with 

```
export BAT_STYLE="header-filename,numbers,grid"
bat Downloads/1.R

```

I got the full path in the header:
`File: Downloads/1.R`

How to trim the dir path, so only keep the file name `1.R `in header? Many thanks!

**Comment by sharkdp** [comment]
That is not possible. `bat` will always show the full path. Using a subshell, you can do
```
(cd Downloads; bat 1.R)
```
and/or potentially add a shell function for this if you need this feature.
